docs/two-phase-commit-transactions.md

```diff
@@ -257,8 +257,8 @@ public class Sample {
         } catch (RollbackException ex) {
           // Rolling back the transaction failed. You can log it here
         }
-      } catch (CrudException | PreparationException | ValidationException | CommitException e) {
-        // If you catch CrudException or PreparationException or ValidationException or
+      } catch (CrudException | PreparationUnsatisfiedConditionException | PreparationException | ValidationException | CommitException e) {
```

Comment:
Ditto.
```suggestion
      } catch (PreparationUnsatisfiedConditionException e) {
        // If you catch PreparationUnsatisfiedConditionException, ...
      } catch (CrudException | PreparationException | ValidationException | CommitException e) {
```

Reply:
Likewise, thank you. 
Fixed in https://github.com/scalar-labs/scalardb/pull/899/commits/46441d23a7772be58cd2ad2bfb887b71767f5b31